pkg/apis/kops/v1alpha1/instancegroup.go

```diff
@@ -69,6 +69,9 @@ type InstanceGroupSpec struct {
 	// AssociatePublicIP is true if we want instances to have a public IP
 	AssociatePublicIP *bool `json:"associatePublicIp,omitempty"`
 
+	// AdditionalSecurityGroupIDs attaches additional security groups (e.g. i-123456)
+	AdditionalSecurityGroupIDs []string `json:"additionalSecurityGroupIDs,omitempty"`
```

Comment:
I think ID is unnecessary...
`AdditionalSecurityGroups []string `json:"additionalSecurityGroups,omitempty"`

Two reasons:

1) We haven't used ID elsewhere (or I'm trying not to)
2) GCE uses `tags` instead, and they are arbitrary strings.  I don't know if we could really reuse the same field, but I feel like it's more likely if we don't have "ID" in there.